core.osgi/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/osgi/commands/TestRepairStatusCommand.java

```diff
@@ -0,0 +1,313 @@
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+import com.ericsson.bss.cassandra.ecchronos.core.Clock;
+import com.ericsson.bss.cassandra.ecchronos.core.metrics.TableRepairMetricsProvider;
+import com.ericsson.bss.cassandra.ecchronos.core.osgi.commands.RepairStatusCommand.OutputData;
+import com.ericsson.bss.cassandra.ecchronos.core.osgi.commands.RepairStatusCommand.SortBy;
+import com.ericsson.bss.cassandra.ecchronos.core.osgi.commands.RepairStatusCommand.Status;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.RepairConfiguration;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.RepairJobView;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.RepairScheduler;
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.RepairStateSnapshot;
+import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReference;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.joda.time.DateTime;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnitParamsRunner.class)
+public class TestRepairStatusCommand
+{
+    private static final long REPAIR_INTERVAL = TimeUnit.DAYS.toMillis(7);
+    private static final long REPAIR_WARNING_TIME = TimeUnit.DAYS.toMillis(8);
+    private static final long REPAIR_ERROR_TIME = TimeUnit.DAYS.toMillis(10);
+
+    private static final OutputData output1 = new OutputData(createTableRef("ks.tbl3"), Status.IN_QUEUE, 0.234, toMillis("1970-01-01T00:00Z"), toMillis("2019-12-31T23:59Z"));
+    private static final OutputData output2 = new OutputData(createTableRef("ks.tbl2"), Status.COMPLETED, 0.0, toMillis("2019-12-24T12:34Z"), toMillis("2019-12-29T00:34Z"));
+    private static final OutputData output3 = new OutputData(createTableRef("ks.tbl1"), Status.WARNING, 1.0, toMillis("2019-11-12T12:34Z"), toMillis("2019-12-29T12:34Z"));
+    private static final OutputData output4 = new OutputData(createTableRef("ks.tbl4"), Status.ERROR, 0.456, toMillis("2029-11-12T23:59Z"), toMillis("2030-11-19T00:00Z"));
+    private static final List<OutputData> outputData = asList(output1, output2, output3, output4);
+
+    private static TableRepairMetricsProvider metricsMock = mock(TableRepairMetricsProvider.class);
+
+    @BeforeClass
+    public static void setup()
+    {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    }
+
+    @Test
+    public void testGetOutputData()
+    {
+        // Given
+        Clock clockMock = mockClock("2019-12-12T12:34");
+        long lastRepairAt = toMillis("2019-12-12T00:00");
```

Comment:
Nit:

Got a bit confused on what the command was printing and had to go back to the class to make sure it did the right thing. (Looking quickly at the test case made me think that we printed repairedAt + error time + REPAIR_INTERVAL, which is not the case)

inQueueTime/warningTime/errorTime could benefit from having a prefix like "repairedAtFor" e.g. "repairedAtForError".